ci/build_docs.sh

```diff
@@ -40,6 +40,14 @@ rapids-mamba-retry install "${PYTHON_CHANNEL}/linux-64/cugraph-dgl-*.tar.bz2"
 export RAPIDS_VERSION_NUMBER="23.12"
 export RAPIDS_DOCS_DIR="$(mktemp -d)"
 
+untar_dest="/tmp/xml_tar"
+for PROJECT in libcugraphops libwholegraph; do
+  rapids-logger "Download ${PROJECT} xml_tar"
+  dest="${untar_dest}/${PROJECT}"
+  mkdir -p "$dest"
```

Comment:
```suggestion
for PROJECT in libcugraphops libwholegraph; do
  rapids-logger "Download ${PROJECT} xml_tar"
  export XML_DIR_${PROJECT^^}=$(mktemp -d)
```
We can save a few lines of code here.

These changes will create arbitrarily named temp directories for each project and store the values in environment variables named `XML_DIR_LIBCUGRAPHOPS` and `XML_DIR_LIBWHOLEGRAPH`.

We can then reference these environment variables in the `conf.py` file.